toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The Good Docs Toolchain
+---
+
+This document summarizes the personae for The Good Docs Toolchain, a collection of tools designed to help users create better technical documentation.
+
+The following are the primary types of users (i.e. personae) targeted by this toolchain. Each has a number of distinct needs, as outlined in the below sections.
+
+# Tech writer
+
+The **Tech Writer** is an individual whose primary role is authoring technical documentation. These may be volunteers in open source projects or employees in commercial organizations. They will typically have a certain level of technical proficiency, although this will range from a simple understanding of technical concepts to full proficiency with tools such as the command line.
+
+As a tech writer...
+
+- I want to use a simple tool to manage docs-as-code so I can concentrate on writing.
+- I want to have access to spell/grammar check so I can take advantage of existing tools.
+- I want to have the option to use the editor of my choice so I don't have to learn something new.
```

Comment:
- I want to ensure consistency across documentation of the content I add/edit, and not have to worry about themes/formatting when I add content.
(This means, say for example, I have to add a table for the page I’m editing. If the rest of the the tables in the documentation are to a specific formatting/styled theme, such should ideally be automatically rendered when I’m adding my table.) - not sure if this point is valid here, but this is a common issue I’ve had to come across when editing pages of an organization in different platforms, like the website, Confluence repo etc.)